dev/com.ibm.websphere.appserver.features/visibility/public/jaxrs-2.2/resources/l10n/com.ibm.websphere.appserver.jaxrs-3.0.properties

```diff
@@ -0,0 +1,19 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables support for Java API for RESTful Web Services v3.0.  JAX-RS annotations \
```

Comment:
assuming we don't care about these since jaxrs-2.2 will never be exposed to users.